src/services/ProcessorService.js

```diff
@@ -24,18 +25,30 @@ async function _fetchSubmissionDetails (submissionId) {
 
   // Fetch member details
   const {
-    createdBy: submitterHandle
+    memberId: submitterId
   } = submission
-  const submitter = await _fetchUserDetails(submitterHandle)
+  const submitter = await _fetchUserDetails(submitterId)
 
+  // Populate communities & community details
+  const challengeESResponse = await helper.apiFetchAuthenticated(`${config.CHALLENGE_API_URL}?filter=id=${submission.challengeId}`)
+  const challengeES = _.get(challengeESResponse, 'result.content[0]', [])
+  const challengeGroups = _.get(challengeES, 'groupIds', []).map(id => id.toString())
+  const challengeCommunities = _.filter(communitiesMetadata, c => _.intersection(c.groupIds, challengeGroups).length > 0)
```

Comment:
This might not be sufficient as there are child groups within other groups. For instance if I have a challenge that's in a child group, I'd have to call out to `/v3/groups/{groupId}/getParentGroup` and iterate till I get to the top to find that ultimate Wipro parent group. I'm wondering if we should try and use the new v5 groups API for this instead. That might be a bit aggressive though.

Reply:
We can fetch all groups from v5 `/v5/groups` and find the top-most group recursively based on the `oldId` property. This endpoint returns the `parentGroups` so there is no need for additional API calls.

Though, it would be great if we could add an additional filter parameter on the `/v5/groups` API to support filtering based on the `challengeId` similar to how it works with `memberId`

Reply:
@ThomasKranitsas `v5/groups` is still in development and can't be used. So we need to use `v3` for now.

cc: @cwdcwd